lib/viewcomponents/port.jsx

```diff
@@ -27,12 +27,15 @@ var Port = React.createClass({
     It's up to the receiver to actually update the backing model as appropriate. 
     If this is not done, the change will revert.
     */
-    onMoveRequested: React.PropTypes.func
+    onMoveRequested: React.PropTypes.func,
+
+    parentScale: React.PropTypes.number
```

Comment:
Why not just `scale` or `zoomScale`? Sure, this is being passed down from the parent, but so are all other props.